docs/reference/modules/node.asciidoc

```diff
@@ -224,8 +235,9 @@ assign data nodes to specific tiers: `data_content`,`data_hot`, `data_warm`,
 `data_cold`, or `data_frozen`. A node can belong to multiple tiers, but a node
 that has one of the specialized data roles cannot have the generic `data` role.
 
+[role="xpack"]
 [[data-content-node]]
-==== [x-pack]#Content data node#
+==== Content data node
 
 Content data nodes accommodate user-created content. They enable operations like CRUD,
 search and aggregations.
```

Comment:
@debadair Is it worth noting that new non-data-stream indices are allocated to the content tier here?
We already mention that in the data tiers docs.